lib/ai/embeddingprocessor_test.go

```diff
@@ -86,25 +93,16 @@ func TestNodeEmbeddingGeneration(t *testing.T) {
 		Jitter:              retryutils.NewSeventhJitter(),
 	})
 
-	done := make(chan struct{})
```

Comment:
@hugoShaka I've removed this channel now that we run the processor throughout the test (and we want to observe changes after modifying nodes). IMO it should be alright to query the calculated embeddings although (no-op) updates might be occurring in the background? The race detector doesn't complain.

Reply:
Yeah, this is very likely a remain from my initial asynchronous implementation. I replaced it with Jakub's one and never adapted the test.